sympy/physics/continuum_mechanics/tests/test_beam.py

```diff
@@ -222,6 +222,24 @@ def test_beam_units():
     assert b.deflection().subs(x, 1*meter) == 62000*meter/(9*E*I)
 
 
+def test_variable_moment():
+    from sympy.functions import log, meijerg
+    from sympy.core.numbers import pi
+    E, I = symbols('E, I', positive=True)
+    x = Symbol('x')
+    b = Beam(4, E, I*(8 - x)/4)
+    b.apply_load(20, 4, -1)
+    R, M = symbols('R, M')
+    b.apply_load(R, 0, -1)
+    b.apply_load(M, 0, -2)
+    b.bc_deflection = [(0, 0)]
+    b.bc_slope = [(0, 0)]
+    b.solve_for_reaction_loads(R, M)
+    assert b.slope().subs(x, 4)*E*I == 4*(-80*log(8) + 80 + 80*log(4))
+    assert b.deflection().subs(x, 4)*E*I == 4*(-960*log(2) - 320*log(4)
+        - 160 - 320*meijerg(((3, 1), ()), ((), (2, 0)), 1) + 640*log(8))
```

Comment:
I'd be nice if you just compared directly to the final expression. Why is meijerg used?

Reply:
`meijerg` appeared due to `integrate` function. This is the same example shown in above description.